### PR TITLE
Implement multi proposal accusation 

### PR DESCRIPTION
**Pull request by vtleonardo**
## Scope

This PR introduces the logic around the multi-proposal accusation of validators.

### CODE REVIEW

File: src/facets/AccusationLibrary.sol [hunk 1]
```diff
@@ -1,40 +1,76 @@
...
+        require(pClaims0.rCert.rClaims.chainId == pClaims1.rCert.rClaims.chainId, "Invalid multiple proposal accusation, the chainId between the proposals are different!");
+
+        // ensure the chainid of blob0 is correct for this chain using RCert sub object of PClaims
+        uint32 chainId = ParticipantsLibrary.participantsStorage().chainId;
+        require(pClaims0.rCert.rClaims.chainId == chainId, "Invalid multiple proposal accusation, the chainId is invalid for this chain!");
+
+        // ensure both accounts are applicable to a currently locked validator - Note<may be done in different layer?>
+        require(ParticipantsLibrary.isValidator(signerAccount0), "Invalid multiple proposal accusation, the signer of these proposals is not a valid validator!");
+
+        return signerAccount0;
```
Comment (anthonygd) [inline src/facets/AccusationLibrary.sol:66]: Why do we return an address?

Comment (vtleonardo) [reply src/facets/AccusationLibrary.sol:66]: If I understood correctly, we need to know the address to the bad validator (who signed the multiple proposals) in order to punish him. The one doing the accusation transaction can be (and probably will be) a different person. Did I get the idea wrong?

File: src/facets/AccusationMultipleProposalFacet.sol [hunk 2]
```diff
@@ -2,18 +2,34 @@
...
+        bytes calldata _signature0,
+        bytes calldata _pClaims0,
+        bytes calldata _signature1,
+        bytes calldata _pClaims1
     ) external {
         AccusationLibrary.AccusationStorage storage s = AccusationLibrary.accusationStorage();
-
-        s.accusations[msg.sender]++;
+        address signer = AccusationLibrary.AccuseMultipleProposal(_signature0, _pClaims0, _signature1, _pClaims1);
+        s.accusations[signer] = s.accusations[signer].add(1);
```
Comment (vtleonardo) [inline src/facets/AccusationMultipleProposalFacet.sol:32]:
```suggestion
        s.accusations[signer] = s.accusations[signer].add(1);
        // TODO: Burn the tokens! Open Question: burn it here or in the AccusationLibrary
```

**Review by anthonygd — APPROVED** [review]
Looks great.

**Pull request merged**